insights/parsers/gluster_vol.py

```diff
@@ -130,20 +130,29 @@ class having a `data` attribute.
     def parse_content(self, content):
         # Stored data in a dictionary data structure
         self.data = {}
-        content = get_active_lines(content, '----')
+
+        # Needed for content with multiple tables
+        content = [line for line in get_active_lines(content, '----') if not line.startswith('There are no active volume tasks')]
```

Comment:
If you just want to ignore this line, a better way is to add it to the `trailing_ingore` list of the `parse_fixed_table`

Reply:
Sure.